src/app/app.module.ts

```diff
@@ -47,7 +47,7 @@ import { SlotProvider } from '../providers/slot/slot';
     CognitoIdentityWrapper,
     ScreenOrientation,
     UrlProvider,
-    SlotProvider
+    SlotProvider,
```

Comment:
if the SlotProvider is only used in the Journal Pages, can we move it to the JournalPageModule.

Reply:
Changed it